lib/Zonemaster/Backend/DB.pm

```diff
@@ -179,6 +183,97 @@ sub _new_dbh {
     return $dbh;
 }
 
+sub _normalize_params {
+    my ( $self, $params ) = @_;
+
+    my $profile = Zonemaster::Engine::Profile->effective;
+
+    my %normalized = ();
+
+    # some of these values are already set in RPCAPI
+    # however setting them here again is required for testing purpose
+    $normalized{domain}   = lc $$params{domain} // "";
+    $normalized{ipv4}     = $$params{ipv4}      // $profile->get( 'net.ipv4' );
+    $normalized{ipv6}     = $$params{ipv6}      // $profile->get( 'net.ipv6' );
+    $normalized{profile}  = $$params{profile}   // "default";
+    $normalized{priority} = $$params{priority}  // 10;
+    $normalized{queue}    = $$params{queue}     // 0;
```

Comment:
@pnax, on a second thought, I do not see why "priority" and "queue" should be part of the fingerprint. It should only be those parameters that can affect the outcome. I do not see how those could affect the outcome.

Reply:
This is a pretty fair point. When I thought about those values, I thought it might avoid mixing results from different queues. But I guess this is not strong enough to support the fact that we use them in the computation of the fingerprint. I guess, we might remove them safely.